en/mkdocs.yml

```diff
@@ -416,69 +416,69 @@ nav:
-          - 'Configure Keystores': deploy/security/configuring-keystores-in-wso2-products.md
-          - 'Renew a CA-Signed Certificate in a Keystore': deploy/security/renewing-a-ca-signed-certificate-in-a-keystore.md
-          - 'Manage Keystores via UI': deploy/security/managing-keystores-with-the-ui.md
-          - 'Add Multiple Keys to the Primary Keystore': deploy/security/adding-multiple-keys-to-the-primary-keystore.md
+          - 'Use Asymmetric Encryption': deploy/security/use-asymmetric-encryption-basic.md
+          - 'Create New Keystores': deploy/security/create-new-keystores.md
+          - 'Configure Keystores': deploy/security/configure-keystores-in-wso2-products.md
+          - 'Renew a CA-Signed Certificate in a Keystore': deploy/security/renew-a-ca-signed-certificate-in-a-keystore.md
+          - 'Manage Keystores via UI': deploy/security/manage-keystores-with-the-ui.md
+          - 'Add Multiple Keys to the Primary Keystore': deploy/security/add-multiple-keys-to-the-primary-keystore.md
         - 'Symmetric Encryption':
           - 'Overview': deploy/security/symmetric-overview.md
           - 'Configurations Related to Symmetric Key Encryption': deploy/security/using-symmetric-encryption.md
     - 'Migrate':
       - 'Migration Process': deploy/migrate/migration-process.md
-      - 'What has Changed': deploy/migrate/migrating-what-has-changed.md
+      - 'What has Changed': deploy/migrate/what-has-changed.md
       - 'Migration Guide':
         - 'Before you Begin': deploy/migrate/migration-guide.md
-        - 'Prepare for Migration': deploy/migrate/migrating-preparing-for-migration.md
-        - 'Migrate to 5.11.0': deploy/migrate/migrating-to-5110.md
-        - 'Migrate Userstore Managers': deploy/migrate/migrating-userstore-managers.md
+        - 'Prepare for Migration': deploy/migrate/prepare-for-migration.md
+        - 'Migrate to 5.11.0': deploy/migrate/migrate-to-5110.md
+        - 'Migrate Userstore Managers': deploy/migrate/migrate-userstore-managers.md
       - 'Feature Deprecation': deploy/migrate/wso2-identity-server-feature-deprecation.md
     - 'Deployment Options':
       - 'AWS': deploy/deployment-options/aws.md
       - 'Docker Desktop': deploy/deployment-options/docker-desktop.md
       - 'EKS': deploy/deployment-options/eks.md
       - 'Minikube': deploy/deployment-options/minikube.md
     - 'Deployment Patterns': deploy/deployment-guide.md
-    - 'Set up WSO2 clusters with Nginx': deploy/fronting-with-the-nginx-load-balancer.md
+    - 'Set up WSO2 clusters with Nginx': deploy/front-with-the-nginx-load-balancer.md
     - 'Set up Separate Databases for Clustering': deploy/settinng-up-separate-databases-for-clustering.md
-    - 'Change the hostname': deploy/changing-the-hostname.md
+    - 'Change the hostname': deploy/change-the-hostname.md
     - 'Introduction to User Management': deploy/introduction-to-user-management.md
   - 'APIs':
     - 'Using APIs': apis/using-apis.md
```

Comment:
There are one or two file names under APIs section that needs to change as well (retrieving-tenant-resources-based-on-search-parameters.md, calling-admin-services.md, using-configuration...") please check